tests/commands/test_list.py

```diff
@@ -29,11 +29,11 @@ async def test_list_events(capsys, generate_models):
     output = capsys.readouterr()
     assert clean_output(output) == [
         "event_0                    WAITING   00000000-0000-0000-0000-000000000000"
-        "    2021-06-06T03:00:00+00:00    2021-06-06T05:00:00+00:00",
```

Comment:
this change is due to the timezone fix. Don't revert it. If this is happening, there must be some problem but the test passes in my machine and in the pipeline. Make sure you're not overriding the timezone somewhere.

Reply:
this change and also all other timestamp related changes are due to [this change](https://github.com/Tech-Workers-Coalition-Italia/mobilizon-reshare/pull/157/files#diff-001e61d97d9dee27d0c5aa1f23ba6c1972cd8a2e8320bac839656b0ab935b84dR9). It was simpler to run tests in UTC, since I found some problem to get timezone aware `Arrow` objects trying to parse ISO formatted strings with `arrow.get`. It appears only that only ISO datetimes